src/xcode/ENA/ENA/Source/HTTPClientCore/Service/Resource/_Resource.swift

```diff
@@ -14,12 +15,15 @@ protocol Resource {
 	var type: ServiceType { get }
 	var sendResource: Send { get }
 	var receiveResource: Receive { get }
-	
+
 	// Defines a default value for no network cases as the specific receive model (for resources like e.g. AppConfig, AllowList)
 	var defaultModel: Receive.ReceiveModel? { get }
-	
+	// Defines the trust evaluation for the certificate pinning.
 	var trustEvaluation: TrustEvaluating { get }
-	
+	// Indicates if the loading of the resource should be retried. Counts descending.
+	// NOTE: Compete logically with defaultModel: When setting both, we will never execute another retry but instead we will return the defaultModel.
```

Comment:
Why is that? Wouldn't it make sense to execute every retry, and if all failed, return the default model?

Reply:
After short discussion, i change the logic so we can define both: default model and retry. After the retries exhaust, we will return the default model.